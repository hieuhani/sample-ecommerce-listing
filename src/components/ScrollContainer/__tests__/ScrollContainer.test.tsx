import React from 'react'
import renderer from 'react-test-renderer'
import { ScrollContainer } from '..'


const props = {
  isFetching: true,
  onLoadMore: () => null,
}
describe('ScrollContainer', () => {
  test('ScrollContainer renders correctly', () => {
    const tree = renderer.create(
      <ScrollContainer {...props}>
        Children
      </ScrollContainer>
    ).toJSON()
    expect(tree).toMatchSnapshot()
  })

})
