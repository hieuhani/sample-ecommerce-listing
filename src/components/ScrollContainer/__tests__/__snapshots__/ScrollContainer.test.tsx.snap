// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ScrollContainer ScrollContainer renders correctly 1`] = `
.c0 {
  overflow-y: auto;
  height: calc(100% - 44pt);
}

<div
  className="c0"
>
  Children
  <p>
    Fetching
  </p>
</div>
`;
